### PR TITLE
fix(client): remove extra 'hours' from /donate page

### PR DESCRIPTION
**Pull request by jmadankumar**
Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

**Comment by jmadankumar** [comment]
Attaching Screenshot
screencapture-freecodecamp-org-donate-2021-02-14-23_03_43

File: client/src/components/Donation/DonateForm.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ class DonateForm extends Component {
   }
 
   convertToTimeContributed(amount) {
-    return `${numToCommas((amount / 100) * 50)} hours`;
+    return `${numToCommas((amount / 100) * 50)}`;
```
Comment (ClockworkOrange605) [inline client/src/components/Donation/DonateForm.js:160]: There is no need to use Template Literals after changes:

`return numToCommas((amount / 100) * 50);`

would be better

**Review by ClockworkOrange605 — CHANGES_REQUESTED** [review]

File: client/src/components/Donation/DonateForm.js (hunk 1 above)
Comment (jmadankumar) [reply client/src/components/Donation/DonateForm.js]: Remove Template Literals.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Thanks @jmadankumar for the PR and @ClockworkOrange605 for the review.

**Pull request merged**